source/_includes/installation/operating_system.md

```diff
@@ -71,46 +76,96 @@ To boot Home Assistant OS, the BIOS needs to have UEFI boot mode enabled and Sec
+Typically, an internal medium like S-ATA hard disk, S-ATA SSD, M.2 SSD, or a non-removable eMMC is used for the x86-64 boot medium. Alternatively, an external medium can be used such as a USB SDD, though this is not recommended.
 
 To write the HAOS image to the boot medium on your x86-64 hardware, there are 2 different methods:
 
-1. Write the HAOS disk image from your desktop computer directly to the boot medium (e.g. using a USB to S-ATA adapter).
-If you can use this method, proceed to "[Write the image to your boot medium](#write-the-image-to-your-boot-medium)" and follow all steps. If you have non-removable internal mediums or don't have the necessary adapter, try the next method instead.
-
-2. Create a "live operating system" on a USB device running e.g. Ubuntu ([how-to guide](https://ubuntu.com/tutorials/try-ubuntu-before-you-install)). Insert it into your system and boot the live operating system. Then follow from step 2 in "[Write the image to your boot medium](#write-the-image-to-your-boot-medium)".
-
-{% details "Ubuntu dependencies for Etcher" %}
+  **Method 1**: Recommended method. It also works on laptops and PCs with internal hard disks. This method boots Ubuntu from a USB flash drive and installs the Home Assistant Operating System from there.
+
+  **Method 2**: With this method, you write the Home Assistant Operating disk image directly onto a boot medium. The steps are a bit more complex. If you have non-removable internal mediums (for example because you are using a laptop) or do not have the necessary adapter (for example an USB to eMMC adapter) use method 1 instead.
+
+### Method 1: Installing HAOS via Ubuntu booting from a USB flash drive
+
+#### Required material
+
+- Computer
+- The target x86-64 hardware, on which you want to install the Home Assistant Operating System (HAOS)
+- USB flash drive (USB thumb drive is sufficient, it should be at least 4&nbsp;GB in size)
+- Internet connection
+
+#### To install HAOS via Ubuntu from a USB flash drive
+
+1. **Notice**: This procedure will write the Home Assistant Operating System onto your device.
+   - This means you will lose all the data as well as the previously installed operating system.
+   - Back up your data before continuing with the next step.
+2. Create a *live operating system* on a USB flash drive:
+   - Follow the ([Ubuntu instructions](https://ubuntu.com/tutorials/try-ubuntu-before-you-install) on writing an Ubuntu iso file onto a USB device.
+3. Insert the USB flash drive into the system on which you want to run Home Assistant.
+   - Boot the live operating system.
+4. When prompted, make sure to select **Try Ubuntu**. This runs Ubuntu on the USB flash device.
+   - The system then starts Ubuntu.
+   - Connect your system to your network and make sure it has internet access.
+5. In Ubuntu, open a browser and open [this procedure](/installation/generic-x86-64).
+   - From there, [download the image][generic-x86-64] to your computer.
```

Comment:
"your computer" could refer the PC you initially started with. I'd drop it to make it "relative" (download to Ubuntu)

```suggestion
   - From there, [download the image][generic-x86-64].
```